src/main/resources/static/js/login.js

```diff
@@ -0,0 +1,96 @@
+class Login extends React.Component {
+    constructor(props) {
+      super(props);
+      this.emailId = this.emailId.bind(this);
+      this.password = this.password.bind(this);
+      this.handleSubmit = this.handleSubmit.bind(this);
+  
+      this.state = {
+        email: '',
+        pass: ''
+      }
+    }
+  
+    emailId(event) {
+      this.setState({ email: event.target.value })
+    };
+    password(event) {
+      this.setState({ pass: event.target.value })
+    };
+  
+  handleSubmit(event){
+    if (this.state.email != "" && this.state.pass != "") {
+      var signupData = {
+        "emailId": this.state.email,
+        "password": this.state.pass
+      };
+  
+      let axiosConfig = {
+        headers: {
+          'Content-Type': 'application/json;charset=UTF-8',
+        }
+      };
+      axios.post('/api/v1/users/signin', signupData, axiosConfig)
+        .then((res) => {
+          if (res.status === 200) {
+            localStorage.setItem('token', res.data.token);
+            localStorage.setItem('mail', res.data.user.emailId);
```

Comment:
Can we set the entire user object in the local storage? setItem('user', res.data.user)
We can may need to find if the logged in user is a manager or not, active or not and so on in the future.